packages/react/src/components/numberInput/NumberInput.test.tsx

```diff
@@ -38,4 +39,30 @@ describe('<NumberInput /> spec', () => {
     const numberInput = screen.getByLabelText('Test label number input', { selector: 'input' });
     expect(numberInput).toHaveValue(0);
   });
+  it('should call onChange when input value is changed directly from input field', async () => {
```

Comment:
A linebreak between test cases would make it more readable, at least in my eyes 👀

Reply:
- [ ] Add a linebreak between test cases